.eslintrc.yml

```diff
@@ -29,7 +29,7 @@ overrides:
       - test/integration/helpers.js
       - lib/growl.js
     parserOptions:
-      ecmaVersion: 2017
+      ecmaVersion: 2018
     env:
```

Comment:
further down there is another one to change.

Reply:
Fix at b63a2ca